x-pack/plugins/security_solution/cypress/e2e/detection_response/rule_management/rules_table/rules_table_auto_refresh.cy.ts

```diff
@@ -34,7 +36,7 @@ import { setRowsPerPageTo } from '../../../../tasks/table_pagination';
 const DEFAULT_RULE_REFRESH_INTERVAL_VALUE = 60000;
 
 // TODO: See https://github.com/elastic/kibana/issues/154694
-describe.skip('Rules table: auto-refresh', () => {
+describe.skip('Rules table: auto-refresh', { tags: [tag.ESS, tag.SERVERLESS] }, () => {
```

Comment:
Same here, since the test is skipped due to being flaky in the past, let's exclude it from serverless